pmlc/dialect/tile/transforms/registration.cc

```diff
@@ -1,13 +1,18 @@
 // Copyright 2020, Intel Corporation
 
+#include "pmlc/dialect/tile/transforms/constant_types.h"
 #include "pmlc/dialect/tile/transforms/contraction.h"
 
 #include "mlir/Pass/Pass.h"
 
 namespace pmlc::dialect::tile {
 
-static mlir::PassRegistration<ComputeBoundsPass> pass(  //
-    "tile-compute-bounds",                              //
+static mlir::PassRegistration<ComputeBoundsPass> compute_bounds_pass(  //
+    "tile-compute-bounds",                                             //
     "Compute bounds for contractions");
 
+static mlir::PassRegistration<ConstantTypesPass> constant_types_pass(  //
+    "tile-constant-types",                                             //
+    "Set constants to specified types");
+
```

Comment:
We'll need to support the ability to pass options for the `floatx` and `intx` values so that we can use this from the `pmlc-opt` command line tool.

Reply:
Added `tile-constant-types-floatx` and `tile-constant-types-intx` command-line options